features/guests_added.feature

```diff
@@ -11,15 +11,46 @@ Feature: See if you are attending an event
       | Other event       | Look, it changes colors     | 22-Apr-3019 | 422 Treeside Way, Berkeley, CA 94704   | stubhub.com/liquid    | 0        |
     And I am on the events page
 
-  Scenario: I register for an event:
+
+
+  Scenario: I register for an event with no demographics:
+    When I follow "rsvp_1"
+    Then I should see "Gender"
+    And I fill out the form with the following attributes:
+      | firstname    | Test           |
```

Comment:
this test should probably include a test that ensures the demographic information is included. might involve changing the view to include some demographic info so that you can test it with cucumber